src/partials/footer.html

```diff
@@ -1,32 +1,43 @@
-<footer>
-  <section class="footer__container">
-    <div class="footer__title">
-      <p>Homemade ice cream made</p>
-      <p>with passion</p>
-    </div>
-    <div class="footer__social-links">
-      <a class="footer__social-icon" href="#">
-        <svg width="20px" height="20px">
-          <use href="*"></use>
-        </svg>
-      </a>
-      <a class="footer__social-icon" href="#">
-        <svg width="20px" height="20px">
-          <use href="*"></use>
-        </svg>
-      </a>
-      <a class="footer__social-icon" href="#">
-        <svg width="20px" height="20px">
-          <use href="*"></use>
-        </svg>
-      </a>
-    </div>
-    <div class="footer__contact">
-      <p class="footer__contact-title">Feel free to contact</p>
-      <a class="footer__contact-phone" href="tel:+61(0) 383 766 284">+61(0) 383 766 284</a>
-    </div>
-    <div class="footer__copy-rights">
-      <p>&#169; 2019 - IceCream. All Rights Reserved.</p>
-    </div>
-  </section>
-</footer>
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+  </head>
+  <body>
+    <footer>
+      <section class="footer container">
+        <div class="footer__title">
+          <p>Homemade ice cream made</p>
+          <p>with passion</p>
+        </div>
+        <div class="footer__social-links">
+          <a class="footer__social-icon" href="#">
+            <svg width="20px" height="20px">
+              <use href="*"></use>
+            </svg>
+          </a>
+          <a class="footer__social-icon" href="#">
+            <svg width="20px" height="20px">
+              <use href="*"></use>
+            </svg>
+          </a>
+          <a class="footer__social-icon" href="#">
+            <svg width="20px" height="20px">
+              <use href="*"></use>
+            </svg>
+          </a>
+        </div>
+        <div class="footer__contact">
```

Comment:
Tutaj "footer__contact" dobrze by było nazwać "footer-contact" będzie to bardziej poprawnie ;)

Reply:
A tutaj bede polemizowac, bo ten kontakt jest elementem sekcji footer, jako blok footer-contact, srednio brzmialby uzyty w innej sekcji. Dlatego opisalam jako Blok__Elemant. Jak to widzisz?

Reply:
Okej, jeśli tak to widzisz to jak najbardziej w porządku :D